### ISSUE TITLE
Request: Toggle Height Limitation by Dimension

### ISSUE THREAD

**Issue by Shahelion** [Tombenpotter/Icarus#16]
There are some mods that add dimensions that...it just doesn't make sense to have the height=fiery death. Perhaps a white/blacklist?

For example:
Deeper Caves, each dimension is a world of caves from 0 to 256. Being able to safely glide down without catching fire would be an amazing asset.
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2350259-deeper-caves-go-beyond-the-void-alpha-3500
Extra Utilities, the Deep Dark. Again, a big cave world.
The Erebus, yet another cave world, full of bugs.

Basically any cave worlds that don't have a visible sky or sun.

**Comment by Tombenpotter** [comment]
I'll add a blacklist, but I'll also add a check to see if you're in direct sunlight or not. :p

**Comment by Shahelion** [comment]
Yeah that's probably the best way to do it. 

What about blocks like Natura's Clouds, or Glass?

Or do those already not block natural sunlight?

**Comment by Tombenpotter** [comment]
What about them?

**Comment by Shahelion** [comment]
Being underneath them still counts as being in sunlight, correct?

**Comment by Tombenpotter** [comment]
Oh, yes, of course
